src/components/InventoryTabs/HybridInventoryTabs.js

```diff
@@ -1,22 +1,35 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import { useLocation } from 'react-router-dom';
 import { Tab, TabTitleText, Tabs } from '@patternfly/react-core';
 import useFeatureFlag from '../../Utilities/useFeatureFlag';
-import useInsightsNavigate from '@redhat-cloud-services/frontend-components-utilities/useInsightsNavigate/useInsightsNavigate';
 import { hybridInventoryTabKeys } from '../../Utilities/constants';
+import { useNavigate } from 'react-router-dom';
```

Comment:
Why does this need to be returned to a raw react router `useNavigate`? 

If it is due to `useInsightsNavigate` not handling states properly than, please wait with replacing it. I am working on extending it to handle these properly.

Reply:
Yup, useInsightsNavigate does not handle states. I would prefer to go with this so that I can continue with my work. Once useInsightsNavigate can handle, then easy to change..